litgpt/utils.py

```diff
@@ -451,9 +451,9 @@ def choose_logger(
     **kwargs: Any,
 ):
     if logger_name == "csv":
-        return CSVLogger(root_dir=(out_dir / "logs"), name="csv", flush_logs_every_n_steps=log_interval, **kwargs)
+        return CSVLogger(root_dir=(out_dir / "logs"), name=name, flush_logs_every_n_steps=log_interval, **kwargs)
```

Comment:
I'd like to point out that "name" that is passed in is not conceptually the same as "name" argument of the logger, it's used differently to make a simplified directory structure. And so the choice here was deliberate (by me). With the change in this PR, there will now be essentially twice the name in the directory structure, which is what I wanted to avoid.